src/dct/ui/cmds.lua

```diff
@@ -281,8 +281,8 @@ function MissionStatusCmd:_mission(_, _, msn)
 	msg = string.format("Mission State: %s\n", msn:getStateName())..
 		string.format("Package: %s\n", msn:getID())..
 		string.format("Timeout: %s (in %d mins)\n",
-			os.date("!%F %Rz", dctutils.zulutime(timeout)),
-			minsleft)..
+			os.date("%F %Rz", dctutils.zulutime(timeout)),
+			minsleft) ..
```

Comment:
You probably do not need to change this line.

Reply:
Accidental, probably caught during a reformatting, will rebase again and fix.